tmail-backend/tmail-third-party/open-ai/src/main/java/com/linagora/tmail/mailet/AIBotMailet.java

```diff
@@ -88,35 +84,35 @@ public void service(Mail mail) {
         }
 
         try {
-            String gptAnswer = askOpenAiModel(mail);
+            String gptAnswer = askChatLanguageModel(mail);
             MimeMessage replyMimeMessage = evaluateReplyMimeMessage(mail, gptAnswer);
 
             getMailetContext()
-                .sendMail(gptAddress, evaluateRecipients(mail), replyMimeMessage);
+                .sendMail(config.getGptAddress(), evaluateRecipients(mail), replyMimeMessage);
         } catch (Exception e) {
             throw new RuntimeException(e);
         }
     }
 
     private boolean gptLoopDetected(Mail mail) {
-        return mail.getMaybeSender().equals(MaybeSender.of(gptAddress));
+        return mail.getMaybeSender().equals(MaybeSender.of(config.getGptAddress()));
     }
 
-    private String askOpenAiModel(Mail mail) throws IOException, MessagingException {
+    private String askChatLanguageModel(Mail mail) throws IOException, MessagingException {
         ChatMessage systemMessage = new SystemMessage("You are a helpful mailing chatbot. Respond to the following email as a chatbot, not as the recipient. ");
         ChatMessage userMessage = new UserMessage(evaluatePrompt(mail));
 
-        return openAiModel.generate(systemMessage, userMessage)
-            .content()
-            .text();
+        return chatLanguageModel.generate(systemMessage, userMessage)
+                .content()
+                .text();
     }
 
     private String evaluatePrompt(Mail mail) throws IOException, MessagingException {
         MessageContentExtractor.MessageContent askingMessageContent = extractMessageContent(mail);
         Optional<String> content = askingMessageContent.extractMainTextContent(htmlTextExtractor);
 
         return Strings.nullToEmpty(mail.getMessage().getSubject()) + "\n" +
-            content.orElse("");
+               content.orElse("");
```

Comment:
idem

Reply:
and for the rest too, maybe check your IDE settings?

Reply:
Please see comment above.